Health App/Home/AddOrUpdateHealthConcernVC.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  AddOrUpdateHealthConcernVC.swift
+//  Health App
+//
+//  Created by Koushik Dutta on 07/10/17.
+//  Copyright © 2017 Virtual Elf LLC. All rights reserved.
+//
+
+import UIKit
+import DataLayer
+class AddOrUpdateHealthConcernVC: UIViewController {
+    
+    @IBOutlet weak var statusPickerView: UIPickerView! = UIPickerView()
+    @IBOutlet weak var titleTextField: UITextField!
+    @IBOutlet weak var statusTextField: UITextField!
+    @IBOutlet weak var noteTextView: UITextView!
+    @IBOutlet weak var outerView: UIView!
+    weak var healthConcern : HealthConcern?
+    let statusOptions = ["In Control","Not In Control","Resolved"]
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        customizedTextView()
+        self.statusTextField.inputView = statusPickerView
+        if let title = healthConcern?.title, !title.isEmpty {
+           titleTextField.isUserInteractionEnabled = false
+            self.title = title
+        }
+        titleTextField.text = healthConcern?.title
```

Comment:
why don't you do a if let for health concern ?

`if let healthConcern = self.healthConcern { then so and so } else { disable delete button etc etc}`